qutebrowser/completion/models/urlmodel.py

```diff
@@ -39,7 +39,11 @@ class UrlCompletionModel(base.BaseCompletionModel):
     def __init__(self, parent=None):
         super().__init__(parent)
 
+        self.columns_to_highlight.append(0)
+        self.columns_to_highlight.append(1)
```

Comment:
No, you can fix this. I think I put it this way because I was using a non-empty list in the base clase, but now the list is empty in the base class, so we can assign instead of pushing.